scripts/listResume.script.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var jobHistories = [];
+
+function JobResume(jobs) {
+  this.jobTitle = jobs.jobTitle;
+  this.empName = jobs.empName;
+  this.empLoc = jobs.empLoc;
+  this.dateEmp = jobs.dateEmp;
+  this.respon = jobs.respon;
+};
+
+JobResume.prototype.toHtml = function() {
+  var $newJobResume = $('jobHistories.listJobs').clone();
```

Comment:
I think this might be where the issue is. Check you selector. This is looking for an element named `jobHistories` instead of a class or id named jobHistories. Try using the most basic selector possible first such as `.listJobs` and see if that does it.